midend/flattenHeaders.cpp

```diff
@@ -19,20 +19,31 @@ limitations under the License.
 
 namespace P4 {
 
+template <typename Func>
```

Comment:
The Annotations class already has a function called 'where' which does exactly this.

Reply:
fixed.

Reply:
please remove this function then.